dist/s_python

```diff
@@ -8,24 +8,24 @@ cd ..
 
 # Check Python coding standards: check for tab characters.
 # Ignore generated files.
-egrep '	' `find . -name '*.py'` |
+egrep '    ' `find . -name '*.py'` |
```

Comment:
Looks like we want to intentionally have tabs here, so that we that we can check against tabs